py_v_sdk/contract/v_stable_swap_ctrt.py

```diff
@@ -0,0 +1,1118 @@
+    )
+
+    def __init__(self, ctrt_id: str, chain: ch.Chain) -> None:
```

Comment:
Perhaps this can be removed as the constructor can be inherited from the base class `Ctrt`